runtime/t0rn-parachain/src/circuit_config.rs

```diff
@@ -125,6 +126,68 @@ impl pallet_clock::traits::OnHookQueues<Runtime> for GlobalOnInitQueues {
     }
 }
 
+parameter_types! {
+    // TODO: update me to be better
+    pub const EscrowAccount: AccountId = AccountId::new([51_u8; 32]);
+    pub const RewardMultiplier: Balance = 1;
+    pub const MinNominatorBond: Balance = 1;
+    pub const MinAttesterBond: Balance = 1;
+    pub const DefaultCommission: Percent = Percent::from_percent(10);
+}
+
+impl pallet_attesters::Config for Runtime {
+    type ActiveSetSize = ConstU32<32>;
+    type BatchingWindow = ConstU32<6>;
+    type CommitmentRewardSource = EscrowAccount;
+    type CommitteeSize = ConstU32<16>;
+    type Currency = Balances;
+    type DefaultCommission = DefaultCommission;
+    type Event = Event;
+    type MaxBatchSize = ConstU32<128>;
+    type MinAttesterBond = MinAttesterBond;
+    type MinNominatorBond = MinNominatorBond;
+    type RandomnessSource = RandomnessCollectiveFlip;
+    type RewardMultiplier = RewardMultiplier;
+    type ShufflingFrequency = ConstU32<400>;
+    type SlashAccount = EscrowAccount;
```

Comment:
Can't we just use `Treasury::account_id()` here? Meaning that slashed funds get dumped back into the treasury.

Reply:
can change 👍